README.md

```diff
@@ -546,11 +558,11 @@ With the cuBLAS backend:
 ```bash
 # Inside <path to onemkl>
 mkdir build && cd build
-export CXX=<path_to_dpcpp_compiler>/bin/dpcpp;
-cmake .. -DENABLE_CUBLAS_BACKEND=True                      \
-         -DENABLE_MKLCPU_BACKEND=False                     \   # disable Intel MKL CPU backend
-         -DENABLE_MKLGPU_BACKEND=False                     \   # disable Intel MKL GPU backend
-         [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \   # required only for testing
+cmake .. -DENABLE_CUBLAS_BACKEND=True  \
+         -DENABLE_MKLCPU_BACKEND=False \                                  # disable Intel MKL CPU backend
+         -DENABLE_MKLGPU_BACKEND=False \                                  # disable Intel MKL GPU backend
+         [-DCMAKE_CXX_COMPILER=<path_to_clang++_compiler>/bin/clang++] \  # required only if clang++ is not found in environment variable PATH
```

Comment:
Could you please move CMAKE_CXX option to the beginning of the command to align it with previous examples?